src/Standards/Generic/Docs/Commenting/DocCommentStandard.xml

```diff
@@ -0,0 +1,268 @@
+    <standard>
+    <![CDATA[
+    The short and long descriptions must start with a capital letter.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Short and long descriptions start with a capital letter.">
+        <![CDATA[
+/**
+ * <em>S</em>hort description.
+ *
+ * <em>L</em>ong description.
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Short or long descriptions don't start with a capital letter.">
+        <![CDATA[
+/**
+ * <em>s</em>hort description.
+ *
+ * <em>l</em>ong description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There must be exactly one blank line separating the short description, the long description and tag groups.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: One blank line separating the short description, the long description and tag groups.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *<em>
+ * Long description.
+<em> *</em>
+ * @param int $foo
+ */
+        ]]>
+        </code>
+        <code title="Invalid: More than one or no blank line separating the short description, the long description and tag groups.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *
+ *
+</em>
+ * Long description.
+ * @param int $foo
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Parameter tags must be grouped together.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Parameter tags grouped together.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+<em> * @param int $foo
+ * @param string $bar</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Parameter tags not grouped together.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * @param int $foo
+<em> *</em>
+ * @param string $bar
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Parameter tags must not be grouped together with other tags.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Parameter tags are not grouped together with other tags.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * @param int $foo
+ *
+<em> * @author Some Author
+ * @return int</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Parameter tags grouped together with other tags.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+<em> * @param  int $foo
+ * @author Some Author
+ * @return int</em>
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Tag values for different tags in the same group must be aligned with each other.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Tag values for different tags in the same tag group are aligned with each other.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * @deprecated <em>1.0.0</em>
+ * @return     <em>bool</em>
+ */
```

Comment:
In a lot of coding standards, the `@return` tag is not allowed to be grouped with other tags, so, even though this sniff does not have any rules about that, we should maybe avoid doing that in the examples. (here and in other places)

What about this instead ?
```suggestion
/**
 * Short description.
 *
 * @since      <em>0.5.3</em>
 * @deprecated <em>1.0.0</em>
 */
```

Reply:
Also implemented in dc22715983ac5a6c76ded1fd5e79e823f619f254